src/cpp/session/modules/SessionCodeTools.R

```diff
@@ -777,8 +777,9 @@
       if (excludeBaseClasses && class %in% c("list", "environment"))
          next
       
-      methodName <- paste(".DollarNames", class, sep = ".")
-      method <- .rs.getAnywhere(methodName, envir = envir)
+      method <- utils::getS3method(".DollarNames", class, optional = TRUE, 
+        envir = getNamespace("utils"))
```

Comment:
Are you sure this should be the `utils` namespace and not the 'active' frame? (can use `.rs.getActiveFrame()` for that)